### DISCUSSION TITLE
Harvest with SAML

### DISCUSSION THREAD

**Discussion by Chris08152** [NetApp/harvest#1648] [Q&A]
As I can see the harvester tries to access the sysmgr. Since SAML is enabled on the filers this access will be redirected to the idP, so Harvester can't login. Is there a workaround for this?

**Comment by rahulguptajss** [comment]
@Chris08152 Could you share the harvest version? Harvest connects directly to filers and not `System Manager`. Could you share the logs from Harvest with the login error?

**Reply by Chris08152** [reply]
I can't excately since I removed the installation. But I remember that a System Manager URL is called at the very begining which is getting redirected to the SAML provider URL. That is where it fails.

**Comment by cgrinds** [comment]
hi @Chris08152 Harvest does not currently support monitoring clusters with SAML enabled. Issue #883 captures this feature request. Can you add a +1 to that issue so we can prioritize? 

As Rahul asked, if you have any logs to share please email them to ng-harvest-files@netapp.com

Thanks!

**Reply by Chris08152** [reply]
I can't excately since I removed the installation. But I remember that a System Manager URL is called at the very begining which is getting redirected to the SAML provider URL. That is where it fails.
